README.md

```diff
@@ -267,6 +267,10 @@ This deployment option does create resources beyond what is created in the ‘St
 
 `cdk bootstrap aws://<aws-account-id>/<aws-region> --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess`
 
+You may need to include the `--trust` option when bootstrapping the production account as shown below. This command is run from the AWS account that has been configured in the property `accountProd` in *packages/threat-composer -infra/cdk.context.json*.
+
+`cdk bootstrap aws://<prod-aws-account-id>/us-west-2 --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess --trust <pipeline-aws-account-id>`
```

Comment:
You may need to include the `--trust` option when bootstrapping the production account as shown below. This command is run from the AWS account that has been configured in the property `accountProd` in *packages/threat-composer -infra/cdk.context.json*.

Reply:
That seems better/more accurate. Have updated README to reflect